configuration/configuration_blackbox_test.go

```diff
@@ -31,7 +40,7 @@ func TestMain(m *testing.M) {
 
 func resetConfiguration() {
 	var err error
-	config, err = configuration.GetConfigurationData()
+	config, err = configuration.NewConfigurationData("../config.yaml")
```

Comment:
I have a question @sbose78. So, any configuration input via a file or ENV variables will be stored in memory(configuration obj), so consequently we'll be able to re-use this config info anywhere. Am I right ?